nvdaHelper/remote/outlook.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+This file is a part of the NVDA project.
+Copyright 2018 NV Access Limited.
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License version 2.0, as published by
+    the Free Software Foundation.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+This license can be found at:
+http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+*/
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <memory>
+#include <comdef.h>
+#include <windows.h>
+#include <common/log.h>
+#include <common/libraryLoader.h>
+#include "inProcess.h"
+#include "nvdaInProcUtils.h"
+
+// The following declarations come from MAPIDEFS.h which is no longer included in the Windows SDK
+
+constexpr ULONG PT_LONG=3;
+constexpr ULONG MAPI_E_NOTFOUND=0x8004010f;
+
+typedef struct {
+	ULONG ulPropTag;
+	ULONG dwAlignPad;
+	union {
+		long l;
+		// other types removed
+	} Value;
+} SPropValue;
+
+using funcType_HrGetOneProp=HRESULT(STDAPICALLTYPE *)(IUnknown*,ULONG,SPropValue**);
+using funcType_MAPIFreeBuffer=ULONG(STDAPICALLTYPE *)(SPropValue*);
+
+// Our RPC function
+error_status_t nvdaInProcUtils_outlook_getMAPIProp(handle_t bindingHandle, const long threadID, IUnknown* mapiObject, const unsigned long mapiPropTag, VARIANT* retVal) {
+	if(!mapiObject) {
+		LOG_ERROR(L"NULL MAPI object");
+		return E_INVALIDARG;
+	}
+	if((mapiPropTag&0xffff)!=PT_LONG) {

```

Comment:
Rather than having `0xffff` hard coded here, it might be better to give it a name. This ties in with the comment in `nvdaInProcUtils` right? Perhaps this could be called `tagMask`